### PR TITLE
LogQL showcase.

### PR DESCRIPTION
**Pull request by weixiang1862**
I renamed existing feature-flag promql to grafana, because logql dashboard depend on promql service and need to share a grafana container with promql.
It is difficult for me to "inject" log dashboard and datasource to an existing promql service. So I renamed it. Now grafana feature flag will publish promql and logql, is it good for you @wankai123 ?

### CODE REVIEW

**Comment by wu-sheng** [comment]
I am good with new feature name. After all, this part of showcase is on grafana.

**Review by kezhenxu94 — APPROVED** [review]
good to me

**Comment by weixiang1862** [comment]
And I need update the ui-grafana doc later. I found grafana will escaped variable value when it enabled `include all` option (I am sorry I missed this case) :
```
6218ddaeaab5409ea4680cea18f8ce3f@172.19.0.2

==>

6218ddaeaab5409ea4680cea18f8ce3f@172\\.19\\.0\\.2
```

Grafana provided [advanced-variable-format-options](https://grafana.com/docs/grafana/latest/dashboards/variables/variable-syntax/#advanced-variable-format-options) to controll this. I changed the loki query expression to this:
```
{service='$service', service_instance='${service_instance:raw}', endpoint='${endpoint:raw}', trace_id='$trace_id'} |= `$keyword_of_content` != `$excluding_keyword_of_content`
```
`raw` format hint grafana do not escape variable value so the service_instance value can keep as it is.

I need update this graph, in case it mislead our user.

**Comment by wu-sheng** [comment]
Feel free to remove the old one and add new screenshot.
The best practice is using a new file name to avoid dns cache(GitHub markdown preview uses that).

**Comment by weixiang1862** [comment]
> Feel free to remove the old one and add new screenshot. The best practice is using a new file name to avoid dns cache(GitHub markdown preview uses that).

Ok.

File: deploy/platform/config/grafana/dashboards/log.json [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+        "type": "datasource"
+      },
+      {
+        "current": {
+          "selected": false
+        },
+        "hide": 2,
+        "includeAll": false,
+        "multi": false,
+        "name": "DS_Prometheus",
```
Comment (wankai123) [inline deploy/platform/config/grafana/dashboards/log.json:298]: I found the DS variable not consistent, could you help unify the name to `DS_SW_Prometheus ` and `DS_SW_Loki `?
And the same as in other promQL dashboards.

Comment (weixiang1862) [reply deploy/platform/config/grafana/dashboards/log.json:298]: Ok.

**Pull request merged**